cogs/file_cog.py

```diff
@@ -11,7 +11,9 @@
 from pdf2image import convert_from_bytes, pdfinfo_from_bytes
 
 
-async def save_jpg_with_target_size(im, filename, target):
+async def save_jpg_with_target_size(
+    im: Image, filename: io.BytesIO, target: int
+) -> io.BytesIO or None:
```

Comment:
This syntax is incorrect. See [here](https://github.com/uwaterloo-tron/discord-bot/blob/96d7e7f07c87342ebc1a3955ad27fe9d32726914/cogs/poll_cog.py#L374-L375) for an example of multiple types.
In Python 3.10.0 you can use `io.BytesIO | None`, but we're using an earlier version of Python so we need to do the older method. Since the second type is None, instead of using `Union` you can use `Optional`:
```py
from typing import Optional

async def save_jpg_with_target_size(
    im: Image, filename: io.BytesIO, target: int
) -> Optional[io.BytesIO]:
```